lib/husky_musher/app.py

```diff
@@ -41,8 +44,12 @@ def handle_unexpected_error(error):
 @app.route('/')
 def main():
     # Get NetID and other attributes from Shibboleth data
-    remote_user = request.remote_user
-    user_info = extract_user_info(request.environ)
+    if DEVELOPMENT_MODE:
+        remote_user = os.environ.get("REMOTE_USER")
```

Comment:
What does the `REMOTE_USER` env variable look like? How do we create one ourselves?

Reply:
It looks like `netid@u.washington.edu`. You create one like any other environment variable: `REMOTE_USER=trsibley@u.washington.edu <command>`.

In production, the value of `REMOTE_USER` comes from Shibboleth [as configured by this field list](https://github.com/seattleflu/backoffice-shibboleth/blob/8e1850f0956b22327804b6666e4e1887ebb39504/shibboleth2.xml#L14).

Reply:
Ah, right, thanks. I didn't dig into this enough, but I wasn't sure if `REMOTE_USER` was the larger object returned by shibboleth. But I see now that's what's returned by the `extract_user_info` function. So never mind. 

Small request: it would be nice to specify, maybe in the README, what the expected value (or format) of the `REMOTE_USER` env variable is in development mode.

Reply:
That's a good request. [I've done so](https://github.com/seattleflu/husky-musher/blob/585ae6a4ad06462c2fd49f1756382d9eb3190f80/README.md#running-locally) in the latest commit.